### PR TITLE
Add Taptic Engine

### PR DESCRIPTION
**Pull request by prettysunflower**

### CODE REVIEW

**Comment by efremidze** [comment]
can u explain more?

**Comment by prettysunflower** [comment]
If you have an iPhone 7/7 Plus, it produce a haptic feedback provided by Taptic Engine. I added it to confirm to the user that the selected index was changed

**Comment by gontovnik** [comment]
Please make it optional and configurable. I believe that not every person who uses it in his project really wants that.

File: DGRunkeeperSwitch/DGRunkeeperSwitch.swift [hunk 1]
```diff
@@ -225,6 +225,9 @@ open class DGRunkeeperSwitch: UIControl {
             UIView.animate(withDuration: animationDuration, delay: 0.0, usingSpringWithDamping: animationSpringDamping, initialSpringVelocity: animationInitialSpringVelocity, options: [UIViewAnimationOptions.beginFromCurrentState, UIViewAnimationOptions.curveEaseOut], animations: { () -> Void in
                 self.layoutSubviews()
                 }, completion: nil)
+            if #available(iOS 10.0, *) {
+                UIImpactFeedbackGenerator(style: UIImpactFeedbackStyle.light).impactOccurred()
+            }
```
Comment (gontovnik) [inline DGRunkeeperSwitch/DGRunkeeperSwitch.swift]: Please make it optional.

**Review by gontovnik — CHANGES_REQUESTED** [review]

File: DGRunkeeperSwitch/DGRunkeeperSwitch.swift (hunk 1 above)
Comment (prettysunflower) [reply DGRunkeeperSwitch/DGRunkeeperSwitch.swift]: Done

**Comment by prettysunflower** [comment]
I made the changes

File: DGRunkeeperSwitch/DGRunkeeperSwitch.swift [hunk 2]
```diff
@@ -59,6 +59,8 @@ open class DGRunkeeperSwitch: UIControl {
         didSet { setNeedsLayout() }
     }
     
+    open var impactFeedbackActivated: Bool = false
```
Comment (gontovnik) [inline DGRunkeeperSwitch/DGRunkeeperSwitch.swift]: Please replace with the `impactFeedbackEnabled` and I will merge it.

**Review by gontovnik — CHANGES_REQUESTED** [review]
Sorry for the delayed response. I am not active on the GitHub this year.

File: DGRunkeeperSwitch/DGRunkeeperSwitch.swift (hunk 2 above)
Comment (prettysunflower) [reply DGRunkeeperSwitch/DGRunkeeperSwitch.swift]: Done

**Pull request closed without merging**